missilewars-plugin/src/main/java/de/butzlabben/missilewars/Config.java

```diff
@@ -45,39 +42,49 @@
     private static final File dir = MissileWars.getInstance().getDataFolder();
     private static final File file = new File(MissileWars.getInstance().getDataFolder(), "config.yml");
     private static YamlConfiguration cfg;
+    private static boolean configNew = false;
 
     public static void load() {
-        boolean configNew = false;
 
+        // check if the directory "/MissileWars" is exist
```

Comment:
Check you grammar, it has to be either "check if ... does exist" or "check if ... exists"